extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -48,6 +54,10 @@ export class BaseSlides extends BaseCarousel {
     this.hasLoop_ = this.element.hasAttribute('loop');
 
     this.hasAutoplay_ = this.element.hasAttribute('autoplay');
+    const autoplayVal = this.element.getAttribute('autoplay');
+    if (autoplayVal != '') {
+      this.autoplayLoops_ = parseInt(autoplayVal, 10);
```

Comment:
`user().assert(isFiniteNumber(this.autoplayLoops_))` would be nice
(`isFiniteNumber` is in `types.js`)